### DISCUSSION TITLE
Trilium server to allow sync. Cheap and simple solution?

### DISCUSSION THREAD

**Discussion by MarcoFaccani** [TriliumNext/Trilium#2570] [Q&A]
Hi,
I was wondering what is the easiest and cheapest option to own Trilium on a server that allows synchronization? I was thinking about AWS EC2 with Docker but I have no idea how expensive it would be (I already used the free year AWS grants).

Thank you for your time and experience

**Comment by zadam** [comment]
Hi, check out this: https://github.com/zadam/trilium/issues/2496
